decidim-core/config/locales/ca.yml

```diff
@@ -202,11 +202,16 @@ ca:
     newsletter_mailer:
       newsletter:
         note: Has rebut aquest missatge perquè estàs subscrit a butlletins de notícies a %{organization_name}. Pots canviar la configuració a la teva <a href="%{link}">pàgina de notificacions</a>.
+        see_on_website: No veus correctament aquest missatge? Mira'l al <a href="%{link}" target="_blank">lloc web</a>.
```

Comment:
Please, only add translations for English locales. We handle translations via Crowdin, and if we add trannslations directly through the code then the integration gets weird.